amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2609,6 +2630,18 @@ void fillDescriptionPtrfromDims(RpptDescPtr &descPtr, vxTensorLayout layout, siz
     }
 }
 
+void fillAudioDescriptionPtrFromDims(RpptDescPtr &descPtr, size_t *tensorDims) {
+    descPtr->n = tensorDims[0];
+    descPtr->h = tensorDims[2];
+    descPtr->w = tensorDims[1];
```

Comment:
Please recheck this
Shouldn't it be like
descPtr->h = tensorDims[1];
descPtr->w = tensorDims[2];

Please omit if tensorDims[2] stores height

Reply:
Check this and address it.

Reply:
h in audio terms is channels and width is samples. Hence this format is correct